### DISCUSSION TITLE
Return vs Enter

### DISCUSSION THREAD

**Discussion by voytxt** [flameshot-org/flameshot#2988] [General]
I [proposed](https://github.com/flameshot-org/flameshot/pull/2987) to change the keyboard shortcut names from `Return` to `Enter` in [README](https://github.com/flameshot-org/flameshot#keyboard-shortcuts).

As [mentioned](https://github.com/flameshot-org/flameshot/pull/2987#discussion_r1025099886) in that pull request, this maybe isn't the best option:

> The `Return` is also something that is used in the config file. I can see you point that on most keyboards it is written "Enter", and the only outlier is Mac, but having Return there is also to some extent educational (because of the config). Perhaps we should find a way to accommodate both, or at least for now revert the change?

What do you guys think?

**Comment by mmahmoudian** [comment]
I created a poll to gauge the public opinion about this: https://fosstodon.org/@flameshot/109359592124481178
The poll will end in 72 hours.
